packages/server/lib/util/suppress_unauthorized_warning.ts

```diff
@@ -0,0 +1,30 @@
+import _ from 'lodash'
+
+let suppressed = false
+
+/**
+ * Don't emit the NODE_TLS_REJECT_UNAUTHORIZED warning while
+ * we work on proper SSL verification.
+ * https://github.com/cypress-io/cypress/issues/5248
+ */
+export function suppress () {
+  if (suppressed) {
+    return
+  }
+
+  suppressed = true
+
+  const originalEmitWarning = process.emitWarning
```

Comment:
move this outside of the `suppress` function so we are the first ones to hold a reference to it

Reply:
moved :+1: